packages/react-day-picker/src/contexts/Focus/getInitialTargetFocusedDay.tsx

```diff
@@ -0,0 +1,38 @@
+import { addDays, endOfMonth, startOfMonth } from 'date-fns';
+
+import { getModifierStatus } from 'hooks/useDayModifiers/utils/getModifierStatus';
+import { Modifiers } from 'types/Modifiers';
+
+export function getInitialTargetFocusedDay(
+  displayMonths: Date[],
+  modifiersContext: Modifiers
+) {
+  const firstDayInMonth = startOfMonth(displayMonths[0]);
+  const lastDayInMonth = endOfMonth(displayMonths[displayMonths.length - 1]);
+
+  let firstEnabledDay = undefined;
```

Comment:
With "enabled" do you mean "focusable"? Clarifying would help:

```suggestion
  let firstFocusableDay = undefined;
```